ru/auth.json

```diff
@@ -0,0 +1,246 @@
+      "confirm_password_input_label": "Подтверждение пароля",
+      "confirm_password_input_error_mismatch": "Укажите идентичные пароли",
+      "continue_button": "Продолжить"
+    },
+    "recovery_codes_page": {
+      "page_title": "Коды восстановления доступа | ${business_name}",
+      "heading": "Почти закончили!",
+      "description": "Скопируйте эти коды восстановления и сохраните их в безопасном месте. Вы сможете войти, если у вас не будет доступа к МФА устройству.",
+      "copy_button": "Скопировать всё",
+      "print_button": "Распечатать",
+      "continue_button": "Продолжить",
+      "download_button": "Загрузить .txt"
+    },
+    "subscribe_form_widget": {
+      "page_title": "Подписаться",
+      "heading": "Подписка на обновления и новости",
+      "description": "Получайте от нас новости о пробукте.",
+      "name_input_label": "Имя",
+      "name_input_error_required": "Аожалуйста введите имя",
+      "name_input_error_exceeds_max_length": "Имя слишком длинное",
+      "email_input_label": "Электронная почта",
+      "email_input_error_invalid": "Ощибка в адресе электронной почты",
+      "email_input_error_required": "Электронная почта обязательна к заполнению",
+      "email_input_error_duplicate_email_found": "Эта электронная почта уже подписана",
+      "continue_button": "Зарегистрироваться",
+      "success_heading": "Успех! Добро пожаловать.",
+      "credit_powered_by": "Работает на"
+    },
+    "password_verify_page": {
+      "page_title": "Проверка пароля",
+      "heading": "Введите ваш пароль",
+      "description": "Введите ваш пароль для ${email_address}",
+      "password_input_label": "Пароль",
+      "password_input_error_required": "Пожалуйста введите ваш пароль",
+      "password_input_error_incorrect": "Пожалуйста введите корректный пароль",
+      "password_input_error_too_short": "Пожалуйста введите пароль длиннее 8 символов",
+      "continue_button": "Продолжить",
+      "cant_sign_in_description": "Не можете войти?",
+      "cant_sign_in_link": "Установить новый пароль"
+    },
+    "sms_verify_code_page": {
+      "page_title": "Мульти-факторная аутентификация | ${business_name}",
+      "heading": "Проверка по номеру телефона",
+      "description": "Введите код, который только что был отправлен на ${phone_number}",
+      "code_input_label": "Код",
+      "code_input_error_expired_code": "Код подтвержения устарел, отправьте новый",
+      "code_input_error_incorrect_code": "Пожалуйста введите полученный код",
+      "code_input_error_incorrect_code_length": "Код состоит из 6 символов",
+      "continue_button": "Проверить",
+      "didnt_receive_a_code_description": "Не получили код?",
+      "didnt_receive_a_code_link": "Послать код ещё раз",
+      "choose_another_method_link": "Выбрать друшой метод"
+    },
+    "verification_code_sms": {
+      "description": "${generated_code} это ваш код для входа в ${business_name} @${business_name} #${generated_code}"
```

Comment:
Hey @evgenyk, @DaveOrDead suggested the second `business_name` (`@${business_name}`) to be `@${receiver_domain}`.